<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>535</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>上传工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>输入目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonOpenDir">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dirEdit">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="dirButton">
       <property name="text">
        <string>浏览</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="labelCount">
           <property name="text">
            <string>local/total</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxBurnIn">
           <property name="toolTip">
            <string>双击时使用burn-in版本(如果有)</string>
           </property>
           <property name="text">
            <string>辅助信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonSelectAll">
           <property name="text">
            <string>全选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonReverseSelection">
           <property name="text">
            <string>反选</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>200</width>
             <height>341</height>
            </rect>
           </property>
           <attribute name="label">
            <string>至服务器文件夹</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item alignment="Qt::AlignLeft">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>服务器</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonOpenServer">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="serverEdit">
                <property name="dragEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="serverButton">
                <property name="text">
                 <string>浏览</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>工程</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="projectEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>文件夹</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="folderEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="epEdit"/>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>集</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="scEdit"/>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>场</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>200</width>
             <height>341</height>
            </rect>
           </property>
           <attribute name="label">
            <string>至CGTeamWork</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkBoxSubmit">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>上传时同时进行一次对应镜头的CGTeamWork提交</string>
              </property>
              <property name="text">
               <string>提交</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>信息</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonClearInfo">
                <property name="text">
                 <string>清空</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTextEdit" name="textEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="syncButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>上传</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="version_label">
     <property name="text">
      <string>version</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionSync">
   <property name="text">
    <string>sync</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionServer">
   <property name="text">
    <string>server</string>
   </property>
  </action>
  <action name="actionDir">
   <property name="text">
    <string>dir</string>
   </property>
  </action>
  <action name="actionUpdateUI">
   <property name="text">
    <string>updateUI</string>
   </property>
  </action>
  <action name="actionOpenDir">
   <property name="text">
    <string>openDir</string>
   </property>
  </action>
  <action name="actionOpenServer">
   <property name="text">
    <string>openServer</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>selectAll</string>
   </property>
  </action>
  <action name="actionReverseSelection">
   <property name="text">
    <string>reverseSelection</string>
   </property>
  </action>
  <action name="actionClearInfo">
   <property name="text">
    <string>clearInfo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dirButton</sender>
   <signal>clicked()</signal>
   <receiver>actionDir</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serverButton</sender>
   <signal>clicked()</signal>
   <receiver>actionServer</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOpenDir</sender>
   <signal>clicked()</signal>
   <receiver>actionOpenDir</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOpenServer</sender>
   <signal>clicked()</signal>
   <receiver>actionOpenServer</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syncButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSync</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonSelectAll</sender>
   <signal>clicked()</signal>
   <receiver>actionSelectAll</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonReverseSelection</sender>
   <signal>clicked()</signal>
   <receiver>actionReverseSelection</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonClearInfo</sender>
   <signal>clicked()</signal>
   <receiver>actionClearInfo</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonClearInfo</sender>
   <signal>clicked()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
